### PR TITLE
Merge 2.1

### PR DESCRIPTION
**Pull request by DavertMik**
Latest changes from 2.1

### CODE REVIEW

File: src/Codeception/Module/Asserts.php [hunk 1]
```diff
@@ -263,35 +262,6 @@ public function assertFileNotExists($filename, $message = '')
     }
 
     /**
-     * @param $expected
```
Comment (Naktibalda) [inline src/Codeception/Module/Asserts.php]: I don't like this change - you removed methods without warning and even without mentioning that in the changelog, but you left aliases that were marked as deprecated long time ago (assertGreaterThen and assertGreaterThenOrEqual ). This is a BC break if anyone is using them.

File: src/Codeception/Module/WebDriver.php [hunk 2]
```diff
@@ -979,8 +983,10 @@ protected function proceedSeeInField(array $elements, $value)
                     break;
 
                 case 'textarea':
+                    // we include trimmed and real value of textarea for check
+                    $currentValues[] = $el->getText(); // trimmed value
```
Comment (Naktibalda) [inline src/Codeception/Module/WebDriver.php]: I think that `break;` is necessary here.

File: src/Codeception/Module/Asserts.php (hunk 1 above)
Comment (DavertMik) [inline src/Codeception/Module/Asserts.php]: yep, you are right. Scrutinzer has suggested that we had those methods written in wrong way

File: src/Codeception/Module/WebDriver.php (hunk 2 above)
Comment (DavertMik) [inline src/Codeception/Module/WebDriver.php]: it's actually not.
$currentValues will contain both: '  text value   ' and 'text value' strings, so both checks will be correct:

```php
$I->seeInField('textarea', ' field value ');
$I->seeInField('textarea', 'field value');
```

**Pull request merged**